specification/azurestack/resource-manager/Microsoft.AzureStack/preview/2020-06-01-preview/examples/EdgeSubscription/ListDeviceIdSubs.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
```

Comment:
>parameters [](start = 3, length = 10)

The examples seems like a duplicate of the other list example and it's not referenced in the swagger.